torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -279,18 +260,14 @@ def reorder_and_filter(inputs, layout_or_out):
                 return [inputs[idx] for idx in input_indices], layout_or_out
 
         def maybe_to_dense(inputs, layout_or_out):
-            int8_gemm = _is_int8_gemm(inputs)
-            wgt_idx = 3 if int8_gemm else 1
+            wgt_idx = 1
```

Comment:
maybe we can define a class-level constant variable for these ids so that we don't have to define it again and again locally?

Reply:
Thanks for the comment and define a class constant of `WGT_IDX=1`. Since `add_choice` is a static method, so we can visit it as `CppPackedGemmTemplate.WGT_IDX` to avoid define it repeatedly.